ingestion/examples/workflows/redash.json

```diff
@@ -1,21 +1,24 @@
 {
   "source": {
     "type": "redash",
-    "config": {
-      "api_key": "api_key",
-      "uri": "http://localhost:5000",
-      "service_name": "redash"
-    }
+    "serviceName": "local_redash",
+    "serviceConnection": {
+      "config": {
+        "type": "Redash",
+        "redashURL": "http://localhost:5000",
+        "apiKey": "api_key"
+      }
+    },
+    "sourceConfig": {"config": {"enableDataProfiler": false}}
```

Comment:
adding enableDataProfiler will take databaseMetadata object
instead, can you add chart and dashboard filter pattern
refer looker or metabase

Reply:
okay I'll add